modules/system/rolls/cast-test.js

```diff
@@ -124,7 +154,13 @@ export default class CastTest extends TestWFRP {
     else // Successful test, casted - determine overcast
     {
       this.result.castOutcome = "success"
-      this.result.description = game.i18n.localize("ROLL.CastingSuccess")
+      this.result.description = game.i18n.localize("ROLL.CastingSuccess");
+      //@/HOUSE
+      if (this.preData.overchannelling > 0) {
+        slOver += this.preData.overchannelling;
+      }
+      //@/HOUSE
```

Comment:
Can you add  `game.wfrp4e.utility.logHomebrew("overchannelling")`

Reply:
done